play sine and starwars theme

luckuisha opened the pull request:
working version

button clicks play sound

luckuisha committed:
play sine and theme with button interaction

dudublad commented on QT/mainwindow.cpp:
```diff
@@ -14,7 +14,20 @@ MainWindow::MainWindow(QWidget *parent)
     setup_FrequencyBox(600);
 }
 
+int tick( void *outputBuffer, void *inputBuffer, unsigned int nBufferFrames,
```
put in mainwindow.h and comment functionality

luckuisha committed:
code cleanup

move declarations to header
change function name for readability
edit stop function

luckuisha committed:
error handling for important sections
